diff -Naur groovy-stubs-orig/main/com/mindalliance/channels/playbook/ifm/Tab.java groovy-stubs/main/com/mindalliance/channels/playbook/ifm/Tab.java
--- groovy-stubs-orig/main/com/mindalliance/channels/playbook/ifm/Tab.java	2008-04-28 16:00:56.000000000 -0400
+++ groovy-stubs/main/com/mindalliance/channels/playbook/ifm/Tab.java	2008-04-28 16:01:52.000000000 -0400
@@ -46,6 +46,7 @@
 	public IModel model(Object object){ throw new Error(); };
 
 	public void remove(Referenceable ref){ throw new Error(); };
+	public void remove(Ref ref){ throw new Error(); };
 
 	/**
 	 * Setter for 'Base' property.
@@ -89,6 +90,7 @@
 	public Object getObject(){ throw new Error(); };
 
 	public Iterator iterator(){ throw new Error(); };
+	public Iterator iterator( int first, int count ){ throw new Error(); };
 
 	public synchronized void detach(){ throw new Error(); };
 
diff -Naur groovy-stubs-orig/main/com/mindalliance/channels/playbook/mem/SessionMemory.java groovy-stubs/main/com/mindalliance/channels/playbook/mem/SessionMemory.java
--- groovy-stubs-orig/main/com/mindalliance/channels/playbook/mem/SessionMemory.java	2008-04-28 16:00:56.000000000 -0400
+++ groovy-stubs/main/com/mindalliance/channels/playbook/mem/SessionMemory.java	2008-04-28 16:00:56.000000000 -0400
@@ -90,6 +90,8 @@
 	 */
 	public void setChanges(Set value){ throw new Error(); };
 
+	public void commit(){ throw new Error(); };
+
 	public Referenceable retrieve(Ref reference){ throw new Error(); };
 
 	public int getPendingDeletesCount(){ throw new Error(); };
diff -Naur groovy-stubs-orig/main/com/mindalliance/channels/playbook/ref/Ref.java groovy-stubs/main/com/mindalliance/channels/playbook/ref/Ref.java
--- groovy-stubs-orig/main/com/mindalliance/channels/playbook/ref/Ref.java	2008-04-28 16:00:56.000000000 -0400
+++ groovy-stubs/main/com/mindalliance/channels/playbook/ref/Ref.java	2008-04-28 16:00:56.000000000 -0400
@@ -22,11 +22,15 @@
  */
 public interface Ref extends Serializable {
 
+	Referenceable deref();
 	Object deref(String path);
 
 	boolean save();
 
+	void remove(Ref ref);
 	void remove(Ref ref, String type);
+	void remove(Referenceable ref);
+	void remove(Referenceable ref, String type);
 
 	void become(Ref ref);
 
@@ -35,6 +39,9 @@
 	Referenceable getReferenced(Store store);
 
 	void add(Ref ref, String type);
+	void add(Ref ref);
+	void add(Referenceable ref, String type);
+	void add(Referenceable ref);
 
 	void changed(String propName);
 
diff -Naur groovy-stubs-orig/main/com/mindalliance/channels/playbook/ref/impl/RefImpl.java groovy-stubs/main/com/mindalliance/channels/playbook/ref/impl/RefImpl.java
--- groovy-stubs-orig/main/com/mindalliance/channels/playbook/ref/impl/RefImpl.java	2008-04-28 16:00:56.000000000 -0400
+++ groovy-stubs/main/com/mindalliance/channels/playbook/ref/impl/RefImpl.java	2008-04-28 16:00:56.000000000 -0400
@@ -32,11 +32,15 @@
 
 	public String db;
 
+	public Referenceable deref(){ throw new Error(); };
 	public java.lang.Object deref(String path){ throw new Error(); };
 
 	public boolean save(){ throw new Error(); };
 
+	public void remove(Referenceable referenceable){ throw new Error(); };
 	public void remove(Referenceable referenceable, String type){ throw new Error(); };
+	public void remove(Ref reference){ throw new Error(); };
+	public void remove(Ref reference, String type){ throw new Error(); };
 
 	public void become(Ref ref){ throw new Error(); };
 
@@ -47,6 +51,9 @@
 	public Referenceable getReferenced(Store store){ throw new Error(); };
 
 	public void add(Referenceable referenceable, String type){ throw new Error(); };
+	public void add(Referenceable referenceable){ throw new Error(); };
+	public void add(Ref referenceable, String type){ throw new Error(); };
+	public void add(Ref referenceable){ throw new Error(); };
 
 	public List drillDown(String listPropName, List drillDownPropNames, Map ddValues){ throw new Error(); };
 
diff -Naur groovy-stubs-orig/main/com/mindalliance/channels/playbook/ref/impl/ReferenceableImpl.java groovy-stubs/main/com/mindalliance/channels/playbook/ref/impl/ReferenceableImpl.java
--- groovy-stubs-orig/main/com/mindalliance/channels/playbook/ref/impl/ReferenceableImpl.java	2008-04-28 16:00:56.000000000 -0400
+++ groovy-stubs/main/com/mindalliance/channels/playbook/ref/impl/ReferenceableImpl.java	2008-04-28 16:00:56.000000000 -0400
@@ -53,6 +53,8 @@
 
 	public void changed(){ throw new Error(); };
 
+	public void changed( String propName ){ throw new Error(); };
+
 	public String getType(){ throw new Error(); };
 
 	public void commit(){ throw new Error(); };
diff -Naur groovy-stubs-orig/main/com/mindalliance/channels/playbook/support/PlaybookSession.java groovy-stubs/main/com/mindalliance/channels/playbook/support/PlaybookSession.java
--- groovy-stubs-orig/main/com/mindalliance/channels/playbook/support/PlaybookSession.java	2008-04-28 16:00:56.000000000 -0400
+++ groovy-stubs/main/com/mindalliance/channels/playbook/support/PlaybookSession.java	2008-04-28 16:00:56.000000000 -0400
@@ -63,6 +63,8 @@
 
 	public Roles getRoles(){ throw new Error(); };
 
+	public void commit(){ throw new Error(); };
+
 	public Ref getProject(){ throw new Error(); };
 
 	public int getPendingDeletesCount(){ throw new Error(); };
