diff -Naur groovy-stubs-orig/main/com/mindalliance/channels/playbook/ref/Ref.java groovy-stubs/main/com/mindalliance/channels/playbook/ref/Ref.java
--- groovy-stubs-orig/main/com/mindalliance/channels/playbook/ref/Ref.java	2008-03-25 09:29:07.000000000 -0400
+++ groovy-stubs/main/com/mindalliance/channels/playbook/ref/Ref.java	2008-03-25 09:29:56.000000000 -0400
@@ -16,6 +16,7 @@
  */
 public interface Ref extends Serializable {
 
+	Referenceable deref();
 	Object deref(String path);
 
 	Ref getReference();
diff -Naur groovy-stubs-orig/main/com/mindalliance/channels/playbook/ref/impl/RefImpl.java groovy-stubs/main/com/mindalliance/channels/playbook/ref/impl/RefImpl.java
--- groovy-stubs-orig/main/com/mindalliance/channels/playbook/ref/impl/RefImpl.java	2008-03-25 09:29:07.000000000 -0400
+++ groovy-stubs/main/com/mindalliance/channels/playbook/ref/impl/RefImpl.java	2008-03-25 09:31:42.000000000 -0400
@@ -24,13 +24,14 @@
  *
  * @source-type GROOVY
  */
-public class RefImpl extends java.lang.Object implements Ref, GroovyInterceptable {
+public class RefImpl extends java.lang.Object implements Ref {
 
 	public String id;
 
 	public String db;
 
 	public java.lang.Object deref(String path){ throw new Error(); };
+	public Referenceable deref(){ throw new Error(); };
 
 	public Ref getReference(){ throw new Error(); };
 
diff -Naur groovy-stubs-orig/main/com/mindalliance/channels/playbook/ref/impl/ReferenceableImpl.java groovy-stubs/main/com/mindalliance/channels/playbook/ref/impl/ReferenceableImpl.java
--- groovy-stubs-orig/main/com/mindalliance/channels/playbook/ref/impl/ReferenceableImpl.java	2008-03-25 09:29:07.000000000 -0400
+++ groovy-stubs/main/com/mindalliance/channels/playbook/ref/impl/ReferenceableImpl.java	2008-03-25 09:29:07.000000000 -0400
@@ -26,7 +26,7 @@
  *
  * @source-type GROOVY
  */
-public class ReferenceableImpl extends java.lang.Object implements Referenceable, GroovyInterceptable {
+public class ReferenceableImpl extends java.lang.Object implements Referenceable {
 
 	public String id;
 
